.github/workflows/build.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
+  workflow_dispatch:
+    branches: [ master ]
```

Comment:
Not needed

Reply:
ok, was just a test as eslint first didn't worked local.

Reply:
I'm fine if you reject this PR. I'm just learning the howl git thing.
From my view there are many more config options needed to be changed. I also thought about sub-config objects. But I think this is the wrong place to discus.

Reply:
I'm fine about adding new features to the module but this module is widely used and introducing such changes is not something I like to do, I prefer an easier approach to keep it simple